src/gui/pedalboard/cables/CableView.cpp

```diff
@@ -113,3 +145,23 @@ void CableView::processorBeingRemoved (const BaseProcessor* proc)
 {
     connectionHelper->processorBeingRemoved (proc);
 }
+
+int CableView::PathGeneratorTask::useTimeSlice()
+{
+    if (cableView.cableBeingDragged())
+    {
+        MessageManager::callAsync (
+            [&]
+            {
+                ScopedLock sl (cableView.cableMutex);
+                if (! cableView.cables.isEmpty())
+                    cableView.cables.getLast()->repaint();
+            });
+    }
+
+    ScopedLock sl (cableView.cableMutex);
+    for (auto* cable : cableView.cables)
+        cable->checkNeedsRepaint();
+
+    return 28; // approx. 35 frames / second
```

Comment:
I've set this so wait a few milliseconds between calls so we leave room for other stuff to run the the time slice, but we could probably bring it down a bit if we want to... just depends on what looks best :).